src/Application.hs

```diff
@@ -12,7 +12,8 @@ module Application
     , handler
     ) where
 
-import Import
+-- TODO: How is threadDelay even exported from there? Should that be changed instead of hiding it here?
+import Import hiding (threadDelay)
```

Comment:
Apparently so. Here is the error message when you don't hide the one from `Import`:

    Ambiguous occurrence ‘threadDelay’
    It could refer to either ‘Import.threadDelay’,
                             imported from ‘Import’ at src/Application.hs:16:1-13
                             (and originally defined in ‘lifted-base-0.2.3.6:Control.Concurrent.Lifted’)
                          or ‘Control.Concurrent.threadDelay’,
                             imported from ‘Control.Concurrent’ at src/Application.hs:21:28-38
                             (and originally defined in ‘GHC.Conc.IO’)